File: NG-Training/NG.NET-Apps/Knowledge-Base-App/src/app/interfaces/assigned-compliance-dto.ts
export interface AssignedComplianceDTO {
    id: string;
    complianceName: string;
    complianceDescription: string;
    requiredPercentage: number;
    createdBy: string;
    createdDate: Date; // or Date if you'll convert it
    questionCount: number;
    assignedDate?: Date | null; // or Date if you'll convert it
    isComplete: boolean;
    completedDate?: Date | null; // or Date if you'll convert it
    complianceScore?: number | null;
}

File: .NET-Training/Backend-Training/StandardApiSolution/KnowledgeBase/Models/DTOs/AssignedComplianceDTO.cs
namespace KnowledgeBaseApi.Models.DTOs
{
    public class AssignedComplianceDTO
    {
        public Guid Id { get; set; }
        public string ComplianceName { get; set; }
        public string ComplianceDescription { get; set; }
        public decimal RequiredPercentage { get; set; }
        public string CreatedBy { get; set; }
        public DateTime CreatedDate { get; set; }
        public int QuestionCount { get; set; }
        public DateTime? AssignedDate { get; set; }
        public bool IsComplete { get; set; }
        public DateTime? CompletedDate { get; set; }
        public decimal? ComplianceScore { get; set; }
    }
}